### PR TITLE
[4.0] Group toolbar buttons for workflow stages and transitions

### PR DESCRIPTION
**Pull request by chmst**
### Summary of Changes

This PR continues the grouping of action buttons in the backend (see #22832, #22393, #23337 and #23336, #23374, #23368, #23380). This time the workflow stages and transitions.

### Testing Instructions
Go to the workflows in the backend and check the toolbar. 
From there you can get stages and transitions. There should be a "Change status" dropdown now which activates when you select one or more workflows.

NOTE: A checkin button will be added as soon as the checked_out functionality is available

### CODE REVIEW

File: administrator/components/com_workflow/View/Stages/HtmlView.php [hunk 1]
```diff
@@ -154,6 +155,8 @@ public function display($tpl = null)
 	protected function addToolbar()
 	{
 		$canDo = ContentHelper::getActions($this->extension, 'workflow', $this->workflowID);
+
```
Comment (Quy) [inline administrator/components/com_workflow/View/Stages/HtmlView.php]:
```suggestion


```

File: administrator/components/com_workflow/View/Transitions/HtmlView.php [hunk 2]
```diff
@@ -153,27 +156,49 @@ protected function addToolbar()
...
-			ToolbarHelper::publishList('transitions.publish');
-			ToolbarHelper::unpublishList('transitions.unpublish');
-		}
-
-		if ($this->state->get('filter.published') === '-2' && $canDo->get('core.delete') && !$isCore)
-		{
-			ToolbarHelper::deleteList(Text::_('COM_WORKFLOW_ARE_YOU_SURE'), 'transitions.delete');
-		}
-		elseif ($canDo->get('core.edit.state') && !$isCore)
+
```
Comment (Quy) [inline administrator/components/com_workflow/View/Transitions/HtmlView.php]:
```suggestion


```

File: administrator/components/com_workflow/View/Stages/HtmlView.php [hunk 3]
```diff
@@ -155,6 +156,9 @@ protected function addToolbar()
 	{
 		$canDo = ContentHelper::getActions($this->extension, 'workflow', $this->workflowID);
 
+
```
Comment (Quy) [inline administrator/components/com_workflow/View/Stages/HtmlView.php]:
```suggestion
```

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 066293656bb527249fce9ef4bc4173f4b4108e48<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23608">issues.joomla.org/tracker/joomla-cms/23608</a>.</sub>

**Comment by bembelimen** [comment]
I have tested this item :white_check_mark: successfully on 066293656bb527249fce9ef4bc4173f4b4108e48<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23608">issues.joomla.org/tracker/joomla-cms/23608</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23608">issues.joomla.org/tracker/joomla-cms/23608</a>.</sub>

**Comment by wilsonge** [comment]
@chmst you need to fix the php cs errors here please

**Comment by brianteeman** [comment]
@wilsonge looks like the changes you requested have all been made

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**